.c9/runners/c.run

```diff
@@ -0,0 +1,18 @@
+// Create a custom Cloud9 runner - similar to the Sublime build system
+// For more information see http://docs.c9.io:8080/#!/api/run-method-run
+{
+    "script": [
+        "base=$file_base_name",
+        "export MODEL=$(awk '{print $NF}' /proc/device-tree/model)",
```

Comment:
Can you add some comments to explain what this does?

Reply:
Done